### PR TITLE
Adds Cough and Sneeze Sounds

### PR DESCRIPTION
**Pull request by Alexshreds**
*cough* What would *cough* you like to *cough* drink today *cough* ma'am? *cough*

:cl: Alexshreds
rscadd: Coughing and sneezing now plays sounds.
/:cl:

### CODE REVIEW

**Comment by Funce** [comment]
I'm actually really afraid of this.

File: code/datums/diseases/advance/symptoms/cough.dm [hunk 1]
```diff
@@ -37,4 +37,4 @@ BONUS
 				var/obj/item/I = M.get_active_hand()
 				if(I && I.w_class == 1)
 					M.drop_item()
-	return
\ No newline at end of file
+	return
```
Comment (tigercat2000) [inline code/datums/diseases/advance/symptoms/cough.dm]: Reeeeeeeeeeeeeeeeeeee

Comment (Alexshreds) [inline code/datums/diseases/advance/symptoms/cough.dm]: what the fuuuuuuuuuuuuuuuuuu

Comment (Alexshreds) [inline code/datums/diseases/advance/symptoms/cough.dm]: 1. I didn't touch this
2. So what?

**Comment by Spacemanspark** [comment]
Sure, why not.

**Comment by Alexshreds** [comment]
well I guess that's one way of doing things >.> <.<

**Comment by Crazylemon64** [comment]
Sick PR brah

**Comment by Krausus** [comment]
These seem to be missing attribution, cooldowns, and female versions. Also, these get kinda weird for some races (IPCs coughing, slime people sneezing, etc).

**Comment by Alexshreds** [comment]
These sounds were meant to be generic, but I realized half of them are indistinguishably male, which I can just split for gender. I'll get around to doing all this tomorrow.

**Comment by jughu** [comment]
Like any girls play ss13 anyways

**Comment by Iamgoofball** [comment]
*cough*
*cough*
vinh'll fix it

**Comment by KasparoVy** [comment]
LMFAO. Been a long time since I've heard that one.

**Comment by Fox-McCloud** [comment]
I'd let everyone sneeze, to be honest; everyone can scream, talk, and an, in fact, use those emotes; it would seem a bit off if they had the emote, but didn't have the actual sound.

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -389,6 +393,10 @@
 				if(!muzzled)
 					message = "<B>[src]</B> coughs!"
 					m_type = 2
+					if(gender == FEMALE)
+						playsound(src, pick(species.female_cough_sounds), 120)
+					else
+						playsound(src, pick(species.male_cough_sounds), 120)
```
Comment (Crazylemon64) [inline code/modules/mob/living/carbon/human/emote.dm]: Won't this `pick from empty list` for slime people or dionae?

**Comment by Crazylemon64** [comment]
Update please!

**Pull request merged**